ballerina/sql_client.bal

```diff
@@ -121,6 +162,10 @@ public client class SQLClient {
         }
     }
 
+    # Performs an SQL `DELETE` operation to delete multiple records from the database.
+    #
+    # + filter - The key-value pairs to be used as the filter (to be used in the SQL `WHERE` clauses)
+    # + return - `()` if the operation is performed successfully or an `error` if the operations fails.
```

Comment:
```suggestion
    # + return - `()` if the operation is performed successfully or an `error` if the operations fail
```

Reply:
Fixed in [83ba8ac](https://github.com/ballerina-platform/module-ballerina-persist/pull/29/commits/83ba8aca943aaaaffd88f25e28fe62a5c00c1ef6)